brion/blueConfig.h

```diff
@@ -74,6 +74,12 @@ class BlueConfig : public boost::noncopyable
                                       const std::string& sectionName,
                                       const std::string& key ) const;
 
+    /**
+     * @return the user and circuit file target parsers.
+     * @version 1.7
+     */
+    BRION_API Targets getTargetParsers() const;
```

Comment:
Yes, getTargets() is more consistent; the real source of the confusion is that all the brion classes define readers/parsers/writers, not the actual objects. If you keep this precondition in mind, then it's fine.